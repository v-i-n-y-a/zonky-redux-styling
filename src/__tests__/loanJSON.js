const loanJSON = {
  id: 1,
  name: 'Pomoc investor≈Ø investorovi',
  story: 'Mala hystorie',
  purpose: '8',
  photos: [
    {
      name: 'file.png',
      url: '/loans/88873/photos/8937',
    },
  ],
  userId: 114025,
  nickName: 'zonky114025',
  termInMonths: 38,
  interestRate: 0.0499,
  rating: 'AAAA',
  topped: null,
  amount: 350000,
  remainingInvestment: 234200,
  investmentRate: 0.33085714285714285,
  covered: false,
  datePublished: '2017-05-14T17:09:54.768+02:00',
  published: true,
  deadline: '2017-05-16T16:50:50.922+02:00',
  investmentsCount: 229,
  questionsCount: 1,
  region: '14',
  mainIncomeType: 'SELF_EMPLOYMENT',
};

describe('<loanJSON object for testing  />', () => {
  it('it it will be changed Layout and ListView test will be changed also', () => {
    expect(loanJSON).toMatchSnapshot();
  });
});

export default loanJSON;
