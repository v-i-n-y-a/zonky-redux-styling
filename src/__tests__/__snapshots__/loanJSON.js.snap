// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<loanJSON object for testing  /> it it will be changed Layout and ListView test will be changed also 1`] = `
Object {
  "amount": 350000,
  "covered": false,
  "datePublished": "2017-05-14T17:09:54.768+02:00",
  "deadline": "2017-05-16T16:50:50.922+02:00",
  "id": 1,
  "interestRate": 0.0499,
  "investmentRate": 0.33085714285714285,
  "investmentsCount": 229,
  "mainIncomeType": "SELF_EMPLOYMENT",
  "name": "Pomoc investor≈Ø investorovi",
  "nickName": "zonky114025",
  "photos": Array [
    Object {
      "name": "file.png",
      "url": "/loans/88873/photos/8937",
    },
  ],
  "published": true,
  "purpose": "8",
  "questionsCount": 1,
  "rating": "AAAA",
  "region": "14",
  "remainingInvestment": 234200,
  "story": "Mala hystorie",
  "termInMonths": 38,
  "topped": null,
  "userId": 114025,
}
`;
